src/Microsoft.PowerShell.ScheduledJob/ScheduledJobTrigger.cs

```diff
@@ -711,7 +711,7 @@ public override bool CanConvertFrom(object sourceValue, Type destinationType)
         {
             if (destinationType == null)
             {
-                throw new ArgumentNullException("destinationType");
+                throw new ArgumentNullException(nameof(destinationType));
```

Comment:
Microsoft.PowerShell.ScheduledJob is not compiled in the repo and we can not accept the changes.